tests/console/autoyast_removed.pm

```diff
@@ -0,0 +1,20 @@
+# SUSE's openQA tests
```

Comment:
please add a reference to the fate https://fate.suse.com/317970 in code. This shouldn't harm but helps to reach the source of information instead of going code > git commit > github PR > progress issue > fate